frontend/src/components/explorer/mapViewer/DataOverlay.vue

```diff
@@ -203,6 +208,9 @@ export default {
       this.errorCustomFileMsg = errorMsg;
       this.showFileLoader = false;
     },
+    validDataTypeInQuery() {
+      return this.$route.query.datatype && Object.keys(this.dataSourcesIndex).indexOf(this.$route.query.datatype) > -1;
+    },
```

Comment:
Does this also validate the dataset?

Reply:
no, only that type `sometype` in `datatype=sometype` is present at our server